.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["es2015-loose", "react"],
-  "plugins": []
+  "presets": ["es2015", "stage-0", "react"],
```

Comment:
Is there a reason for `stage-0` use? That stage is typically reserved for experimental features that might not make it out of committee.